framework/platform/CMakeLists.txt

```diff
@@ -43,6 +43,17 @@ if (NOT DEFINED TCUTIL_PLATFORM_SRCS)
 		if (NOT (DEQP_SUPPORT_EGL OR DEQP_SUPPORT_GLX))
 		  message(FATAL_ERROR "At least one of EGL and GLX must be enabled for X11")
 		endif ()
+
+	elseif (DEQP_USE_WAYLAND)
+		set(TCUTIL_PLATFORM_SRCS
```

Comment:
as your comments, I have referenced `targets/raspi/raspi.cmake'   and I  have updated `wayland.cmake'. so current, `wayland.cmake' sets TCUTIL_PLATFORM_SRCS.